VadimVildanov/HW7/Task3.py

```diff
@@ -0,0 +1,22 @@
+# I create function with argument "word". "word" is word which will enter user.
+def calculates_characters(word):
+    # I decided do with dict and save letters
+    dict = {}
+    # Iterate through each letters in "word"
+    for letters in word:
```

Comment:
look at this version
def calculate_characters(word):
    character_count = {}
    
    # Iterate through each letter in "word"
    for letter in word:
        # Use get method to get the current count or default to 0 if not present
        current_count = character_count.get(letter, 0)
        
        # Increment the count
        character_count[letter] = current_count + 1

    return character_count

string = input("Enter your word: ")
print(calculate_characters(string))
def calculate_characters(word):
    character_count = {}
    
    # Iterate through each letter in "word"
    for letter in word:
        # Use get method to get the current count or default to 0 if not present
        current_count = character_count.get(letter, 0)
        
        # Increment the count
        character_count[letter] = current_count + 1

    return character_count

string = input("Enter your word: ")
print(calculate_characters(string))